lang/c++/impl/json/JsonIO.hh

```diff
@@ -404,9 +404,25 @@ public:
 
     template<typename T>
     void encodeNumber(T t) {
+        sep();
+        const std::string s = std::to_string(t);
+        out_.writeBytes(reinterpret_cast<const uint8_t *>(s.data()), s.size());
+        sep2();
+    }
+
+    void encodeNumber(float t) {
```

Comment:
Since this is a specialization, it'll better to be explicit about it. Please add: `template <>` before this line.